framework/src/Jovo.ts

```diff
@@ -408,6 +421,37 @@ export abstract class Jovo<
     return this.$session.isNew;
   }
 
+  getPersistableData(): JovoPersistableData {
+    return JSON.parse(
```

Comment:
I am assuming JSON.stringify followed by JSON.parse is used to get rid of references and get a deep copy?
Or is it also used to get rid of functions and class-instances?

In case it's just used for getting a deep copy we could either use `lodash.clonedeep`, which is a dependency already, or make sure that `getPersistableData` returns a copy already and clone the `history` via `slice` although this would not deep-clone the items of `history`. But for that issue, a mapper could be used.

Reply:
Thanks, removed stringify->parse. It wasn't necessary here.